### PR TITLE
Update image component

### PR DESCRIPTION
**Pull request by hstandaert**
## Description

* Removed figure component and merged it into `Image`
* Update image placeholder according to design (https://zpl.io/VxBpDpx)
* Added broken image placeholder

## Screenshots (if appropriate):
screencapture-localhost-3000-components-preview-image-2019-08-29-08_10_11

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

**Review by delrueba — CHANGES_REQUESTED** [review]
What about the figure documentation?

**Comment by hstandaert** [comment]
> What about the figure documentation?

Good point! I added it to the `Image` component. @mattiasberlamont, could you review whether this documentation is sufficient for the Image component?

File: components/21-atoms/image/README.md [hunk 1]
```diff
@@ -19,5 +17,7 @@ For images, the following requirements should be followed:
 
 The image component is used in the:
 
+* <a href="{{path './teaser.html'}}">Teaser component</a>
+* <a href="{{path './teaser.html'}}">Teaser component</a>
```
Comment (mattiasberlamont) [inline components/21-atoms/image/README.md]:
```suggestion
* <a href="{{path './header.html'}}">Header component</a>
```

**Comment by mattiasberlamont** [comment]
@hstandaert I could not find how to do suggestions in a proper way, so I updated the documentation myself and added a commit.

Another suggestion based on the screenshot (but I cannot change it myself):

* I would rename the variation "placeholder" to "no image placeholder".
* I would rename the variation "broken" to "broken image placeholder".

This to make clear which placeholder should be used when.

I refer to these placeholders in the updated Image documentation.

**Comment by hstandaert** [comment]
> @hstandaert I could not find how to do suggestions in a proper way, so I updated the documentation myself and added a commit.
> 
> Another suggestion based on the screenshot (but I cannot change it myself):
> 
> * I would rename the variation "placeholder" to "no image placeholder".
> * I would rename the variation "broken" to "broken image placeholder".
> 
> This to make clear which placeholder should be used when.
> 
> I refer to these placeholders in the updated Image documentation.

@mattiasberlamont, thanks for you contribution. I have updated the naming of the variants as suggested 🙂

**Review by delrueba — APPROVED** [review]

**Pull request merged**